index.css

```diff
@@ -30,16 +30,34 @@ h1{
     font-weight: bold;
 }
 button {
-    background: #3d9cb1;
```

Comment:
I think I prefer the darker background for the 3 primary buttons. Do you want to see if you can find a hover state that works well with keeping the darker background on the initial state?

Reply:
Actually, I don't understand this well. Do you mean you don't want those 3 buttons to be transparent?
I really have no idea how to choose colors for the initial state and the hover state. If you suggest the colors I will fix it. 

*By the way, in my opinion, when all the buttons are transparent it has a nice and unified look. But it's your code, so you decide :)

Reply:
I meant basically make the :hover the opposite of what it is now. So the button is solid to start and transparent on hover. Not sure if that will look good but I feel the transparent is a little hard to read

Reply:
Ok, now I understand. Sorry, English is my 3rd language and I'm not very good at it.
Anyway, I fixed it. See if it's what you want?